Sectors/Lines - LACC.txt

```diff
@@ -511,48 +511,65 @@ COORD:N051.29.60.000:E002.00.00.000
 COORD:N051.27.20.000:E002.30.00.000
 
 SECTORLINE:SAS3
-DISPLAY:London AC Clacton:London AC Clacton:EBBU
-DISPLAY:London TC East:London TC East:EBBU
-COORD:N051.38.13.000:E002.29.60.000
+DISPLAY:London AC Clacton:London AC Clacton:EHAA
+DISPLAY:London TC East:London TC East:EHAA
+COORD:N051.38.11.510:E002.29.59.000
 COORD:N051.27.20.000:E002.30.00.000
 
 SECTORLINE:REF1
 DISPLAY:London AC Clacton:London AC Clacton:EHAA
 DISPLAY:London AC Clacton:London AC Clacton:London TC East
-DISPLAY:London TC East:London TC East:London AC Clacton
-COORD:N051.57.02.000:E002.21.23.000
-COORD:N051.42.45.000:E002.10.01.000
+DISPLAY:London TC East:London TC East:EHAA
+COORD:N051.42.44.830:E002.10.01.170
+COORD:N051.57.02.160:E002.21.22.620
 
 SECTORLINE:REF2
 DISPLAY:London AC Clacton:London AC Clacton:EHAA
+DISPLAY:London AC Clacton:London AC Clacton:London TC East
 DISPLAY:London TC East:London TC East:EHAA
```

Comment:
Again the logic here doesn't make sense. To the East of REF2, all airspace belongs to EGTT, so we're just worried about internal sectors i think.